src/components/Popover/types.ts

```diff
@@ -107,7 +107,8 @@ export type PopoverDefaultProps = {
     size: 's' | 'l';
 };
 
-export type PopoverProps = PopoverExternalProps &
+export type PopoverProps = Omit<PopperProps, 'offset' | 'modifiers'> &
```

Comment:
What if new properties are added to the `PopperProps`?  Typescript will think that `Popover` supports them, but, in fact, it won't.

Reply:
I agree with the remark, I will correct it. But in general Popover should support all Popup properties in my opinion